### ISSUE TITLE
Feature Request: Google Analytics profile linking

### ISSUE THREAD

**Issue by timnovis** [rhyswfbowles/SeoCheck#4]
If it's possible, as a user, it would be great to automatically link and import profiles from Google Analytics

**Comment by rhyswfbowles** [comment]
Yeah this is definitely something that would make a great additon
